zipline/finance/performance/position_tracker.py

```diff
@@ -350,14 +350,16 @@ def get_positions(self):
                     pass
                 continue
 
-            # Note that this will create a position if we don't currently have
-            # an entry
-            position = positions[sid]
+            position = zp.Position(sid)
```

Comment:
Ok great.  Sounds like we should add last_sale_date to the init then.  Or maybe consolidate with https://github.com/quantopian/zipline/blob/16fd6681a66101d3f06a0bb639ee699a8905ceda/zipline/finance/performance/position.py#L43-L43?  I'm a bit confused on when each is used.